tests/modules/nf-core/krakenuniq/preloadedkrakenuniq/test.yml

```diff
@@ -4,24 +4,22 @@
     - krakenuniq/preloadedkrakenuniq
     - krakenuniq
   files:
-    - path: output/krakenuniq/test.classified_1.fastq.gz
-    - path: output/krakenuniq/test.classified_2.fastq.gz
     - path: output/krakenuniq/test.krakenuniq.classified.txt
     - path: output/krakenuniq/test.krakenuniq.report.txt
-    - path: output/krakenuniq/test.unclassified_1.fastq.gz
-    - path: output/krakenuniq/test.unclassified_2.fastq.gz
+    - path: output/krakenuniq/test.merged.classified.fasta.gz
```

Comment:
Don't want to take a stab at converting to nf-test? :grimacing: https://nf-co.re/docs/contributing/modules#migrating-from-pytest-to-nf-test

Reply:
Hmm, maybe.

Reply:
Well, wasn't so hard after all. (Famous last words.)